bliss/surveys/sdss.py

```diff
@@ -345,12 +378,157 @@ def restrict_by_ra_dec(self, ra_lim, dec_lim):
             d[key] = val[:, keep]
 
         return PhotoFullCatalog(
-            plocs[0, :, 0].max() - plocs[0, :, 0].min(),  # new height
-            plocs[0, :, 1].max() - plocs[0, :, 1].min(),  # new width
+            int(plocs[0, :, 0].max() - plocs[0, :, 0].min()),  # new height
+            int(plocs[0, :, 1].max() - plocs[0, :, 1].min()),  # new width
             d,
         )
 
 
+class SDSSPointSpreadFunction(ImagePointSpreadFunction):
```

Comment:
It's pretty universal the people call a point spread function a "PSF". So perhaps `ImagePSF` as a class name instead?

Reply:
Done.